### ISSUE TITLE
Only one aspect can be predicted? 

### ISSUE THREAD

**Issue by TeTeTang** [teapot123/JASen#4]
Hi, I noticed the sample test set only has one aspect/sentiment labeled for each review/doc, but the paper says your model aims to output a set of joint topics for each review, how to adjust the test set to achieve that goal? Thanks.

**Comment by teapot123** [comment]
Hi, thanks for your interest! The setting of my paper is to predict the sentiment and aspect simultaneously for each review sentence, so it is still a single-label problem. My method may not work well on multi-label problems (if I understand you correctly), but since it outputs prediction logits for all the aspects, perhaps you can try to set a threshold to get multiple aspects for each review. However, it would still be better to use loss functions designed specifically for multi-label classification to deal with this problem.

**Comment by TeTeTang** [comment]
Thanks for the response, that resolved my question.
